webapp/channels/src/components/suggestion/suggestion_date.tsx

```diff
@@ -54,7 +58,6 @@ const SuggestionDate = ({
         >
             <Component
                 key={term}
-                ref={term}
```

Comment:
This is the real change that fixes the crash. The rest are to remove warnings and minor improvements.